pulp_rpm/app/modelresource.py

```diff
@@ -80,6 +80,23 @@ class ModulemdResource(RpmContentResource):
     Resource for import/export of rpm_modulemd entities.
     """
 
+    packages = fields.Field(
+        column_name="package_ids",
+        attribute="packages",
+        widget=ManyToManyWidget(model=Package, separator=",", field="pkgId"),
+    )
+
+    def before_import_row(self, row, row_number=None, **kwargs):
+        if "packages" in row:
```

Comment:
```suggestion
        super().before_import_row(row, **kwargs)

        if "packages" in row:
```

@hao-yu sorry it took so long to get back to this. The domains-work takes advantage of before_import_row(), so any override needs to let super() have a shot at the row first. With this change, the PR looks to work fine!

Reply:
@ggainey no worries. :)

I have made the suggested change.